.github/workflows/linting.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: WPCS check
-        uses: 10up/wpcs-action@stable
+        uses: 10up/wpcs-action@72a56d82dcf853abe99ea617e96a42ce9c2c175a # Merge of PHP deprecation fix.
```

Comment:
@peterwilsoncc this commit is part of the stable branch, should we close this PR?

Reply:
@Sidsector9 Done.